src/main/java/services/MenuSessionRunnerService.java

```diff
@@ -371,21 +370,13 @@ private static StackFrameStep getStepToFrame(SessionWrapper session) {
         return stepToFrame;
     }
 
-    private static void setPersistentCaseTile(MenuSession menuSession, NewFormResponse formResponse) {
```

Comment:
got rid of these because they seemed pointless (one-line helpers with the same name as the functions they call)

Reply:
:+1: very nice, thanks